src/main/webapp/resources/js/pages/projects/processing/ProcessingCoverage.jsx

```diff
@@ -0,0 +1,161 @@
+        return Promise.resolve();
+      }
+      return Promise.reject("Must be a numeric value");
+    },
+  });
+
+  const update = () =>
+    form.validateFields().then((values) => {
+      updateProcessingCoverage(projectId, values)
+        .then((message) => {
+          setVisible(false);
+          setCoverage(values);
+          notification.success({ message });
+        })
+        .catch((message) => {
+          setVisible(false);
+          notification.info({ message });
+        });
+    });
+
+  return (
+    <section>
+      <div
+        style={{
+          display: "flex",
+          justifyContent: "space-between",
+          alignItems: "center",
+        }}
+      >
+        <Typography.Title level={3}>
+          {i18n("ProcessingCoverage.title")}
+        </Typography.Title>
+        {canManage && (
+          <span>
+            <Button onClick={() => setVisible(true)}>
+              {i18n("form.btn.edit")}
+            </Button>
+            <Modal
+              title={i18n("ProcessingCoverage.modal.title")}
+              visible={visible}
+              onCancel={() => setVisible(false)}
+              onOk={update}
+            >
+              <Form
+                layout="vertical"
+                initialValues={{
+                  minimum: isNumeric(coverage.minimum)
+                    ? coverage.minimum
+                    : null,
+                  maximum: isNumeric(coverage.maximum)
+                    ? coverage.maximum
+                    : null,
+                  genomeSize: isNumeric(coverage.genomeSize)
+                    ? coverage.genomeSize
+                    : null,
+                }}
+                form={form}
+              >
+                <Form.Item
+                  label={i18n("ProcessingCoverage.minimum")}
+                  name="minimum"
+                  precision={0}
+                  rules={[numericValidator]}
+                >
+                  <InputNumber style={{ width: `100%` }} step={100} min={0} />
+                </Form.Item>
+                <Form.Item
+                  label={i18n("ProcessingCoverage.maximum")}
+                  name="maximum"
+                  rules={[numericValidator]}
+                >
+                  <InputNumber style={{ width: `100%` }} step={100} min={0} />
+                </Form.Item>
+                <Form.Item
+                  label={i18n("ProcessingCoverage.genomeSize")}
+                  name="genomeSize"
+                  rules={[numericValidator]}
+                >
+                  <InputNumber style={{ width: `100%` }} step={100} min={0} />
+                </Form.Item>
+              </Form>
+            </Modal>
+          </span>
+        )}
+      </div>
+      <Row gutter={16}>
+        <Col span={8}>
+          <Card>
+            <Statistic
+              title={i18n("ProcessingCoverage.minimum")}
+              value={isNumeric(coverage.minimum) ? coverage.minimum : "Not set"}
```

Comment:
Internationalize the "Not Set" here and below

Reply:
Updated in 23a2f08